modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -14,5 +14,5 @@ Sensor also triggers deploy-time and runtime violations based on {product-title-
 * *Admission controller*: The Admission controller prevents users from creating workloads that violate security policies in {product-title-short}.
 * *Collector*: Collector analyzes and monitors container activity on cluster nodes.
 It collects container runtime and network activity information and sends the collected data to Sensor.
-* *Scanner*: On Kubernetes, the Secured cluster services do not include Scanner.
-However, on the {ocp}, {product-title-short} installs a lightweight Scanner version on each secured cluster to scan images in the integrated {ocp} registry.
+* *Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
+However, on the {ocp}, {product-title-short} installs a lightweight Scanner-slim version on each secured cluster to scan images in the integrated {ocp} registry.
```

Comment:
I believe the terms `lightweight Scanner` and `Scanner-slim` can be used interchangeably, thoughts on using `lightweight Scanner` everywhere instead of `Scanner-slim`?

Reply:
We have decided to use Scanner-slim at all instances.